plugin-server/src/worker/ingestion/process-event.ts

```diff
@@ -658,6 +747,41 @@ export class EventsProcessor {
         return data
     }
 
+    private async createPerson(
```

Comment:
I initially needed this because of identify, but similar to what Karl did better to have less logic in the db file, so kept this change.